src/shims/unix/freebsd/foreign_items.rs

```diff
@@ -47,6 +47,21 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                     this.read_scalar(len)?,
                 )?;
             }
+            "getrandom" => {
+                let [ptr, len, _flags] =
```

Comment:
This variable is used now, please rename it to `flags`.

Reply:
ok

Reply:
wait ... now the build complains if I rename it.